tests/serverpod_test_server/migrations/migration_registry.txt

```diff
@@ -5,3 +5,4 @@
 ### the conflict by removing and recreating the conflicting migration.
 
 20240125141253049
+20240520103319212
```

Comment:
Discussion: Would be good if we have a single migration for each major version update of Serverpod maybe?

Reply:
Probably yes.